frontend/www/js/omegaup/problem/collection.ts

```diff
@@ -0,0 +1,25 @@
+import Vue from 'vue';
+import problem_collections from '../components/problem/Collection.vue';
```

Comment:
nit:

```suggestion
import problem_Collection from '../components/problem/Collection.vue';
```

la convención es `<nombre_del_directorio>_<NombreDeLaClase>`.